src/platform/forms-system/src/js/widgets/SelectWidget.jsx

```diff
@@ -51,6 +57,8 @@ function SelectWidget({
         const newValue = getValue(event, multiple);
         onChange(processValue(schema, newValue));
       }}
+      {...widgetProps}
+      {...(value && selectedProps[value]) || {}}
```

Comment:
What is this doing?

Reply:
It's similar to what the radio widget is doing, showing a dynamic prop when a specific item is selected. If an `aria-describedby` points to an ID that doesn't exist, but shows up when a value is selected, it will cause an axe error regarding an ID not found in the DOM.